modules/yardstick/src/main/java/org/apache/ignite/yardstick/cache/IgniteGetTxExplicitBenchmark.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.yardstick.cache;
+
+import java.util.Map;
+import java.util.concurrent.Callable;
+import org.apache.ignite.IgniteCache;
+import org.apache.ignite.IgniteTransactions;
+import org.apache.ignite.yardstick.IgniteBenchmarkUtils;
+import org.apache.ignite.yardstick.cache.model.SampleValue;
+import org.yardstickframework.BenchmarkConfiguration;
+
+/**
+ * Ignite benchmark that performs get operation inside transaction.
+ */
+public class IgniteGetTxExplicitBenchmark extends IgniteCacheAbstractBenchmark<Integer, Object> {
+    /** IgniteTransactions. */
+    private IgniteTransactions txs;
+
+    /** Get operation callable. */
+    private Callable<Void> clo;
```

Comment:
Rename to  `c` or `call` according to [abbreviation rules](https://cwiki.apache.org/confluence/display/IGNITE/Abbreviation+Rules). `clo` is very rarely used in the project.

Reply:
fixed